### ISSUE TITLE
Maybe possible

### ISSUE THREAD

**Issue by xdSpoty** [phoenixthrush/AniWorld-Downloader#27]
Hey, maybe its possible to ad languege keys to the file name like 'One Piece - S1E10.ja.mp4' or 'One Piece - S1E10.en.mp4 this can be good for manage downloaded content or for Plex to manage langueges.

And a other idea is a bulk download button to download all available episodes at ones.

Thx for your work, i have self made a aniworld downlaoder but yours is better.

**Comment by phoenixthrush** [comment]
Hi,

bulk download is already possible by using command line arguments and appending --keep-watching or by using an episode file. If you need help, please look at the README for that.

The lang key in filename is a good idea, I'll add in a moment.

Thank you for the kind words! :)

**Comment by phoenixthrush** [comment]
I've just published v.2.4.2 with the added language in filename.
`pip install -U aniworld` to update. :)

**Comment by phoenixthrush** [comment]
Do you still need help?  
Does it work now with the added language in the filename when using Plex, or do I need to change the naming scheme?

**Comment by xdSpoty** [comment]
According to the internet you should make filename S1E1.en.mp4 but I don't think plex can handle it because the sound is unknown so it doesn't recognize it by filename. But your way is also good.

![grafik](https://github.com/user-attachments/assets/6cfc59b9-5611-4c37-af8c-752fe61a22ba)

**Comment by phoenixthrush** [comment]
The issue is that if the audio is in Japanese and I name it .jp, the subtitles could still be in German or English. If you have a better idea for handling this in Plex that works fine, let me know! :O

**Comment by xdSpoty** [comment]
Yes that's right i hadn't really thought about the fact that even with english subtitle the sound is jap.

i only found this thread and someone ask for anime solution but not even plex has a solution.
https://www.reddit.com/r/PleX/s/9lkeVbW29J

the only way to get subs in plex is with the sub txt and the blank anime

**Comment by phoenixthrush** [comment]
Yeah, aniworld doesn’t have japanese audio without subtitles. We could add another site for that and include a subtitle fetcher, but something like that would be implemented in the `next` branch anyways.

**Comment by xdSpoty** [comment]
for me there is no need for this feature for now, i just made two libs in plex one with sub one with dub

**Comment by phoenixthrush** [comment]
That's cool, I'll close this issue then if it's okay. If you need help with anything again, don't hesitate to create a new issue. >_<
